prime-router/settings/STLTs/MN/mn-doh.yml

```diff
@@ -0,0 +1,101 @@
+# To submit this to localhost, run:
+#
+# ./prime multiple-settings set --input ./settings/STLTs/MN/mn-doh.yml
+#
+- name: "mn-doh"
+  description: "Minnesota Department of Health"
+  jurisdiction: "STATE"
+  stateCode: "MN"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "mn-doh"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation: !<HL7>
+        schemaName: "azure:/hl7_mapping/receivers/STLTs/MN/MN-receiver-transform.yml"
+        useTestProcessingMode: false
+        useBatchHeaders: true
+        receivingApplicationName: "MEDSS-ELR"
+        receivingApplicationOID: "2.16.840.1.114222.4.3.3.6.2.1"
+        receivingFacilityName: "MN DOH"
+        receivingFacilityOID: "2.16.840.1.114222.4.1.3661"
+        messageProfileId: null
+        replaceValue: {}
+        replaceValueAwithB: {}
+        reportingFacilityName: null
+        reportingFacilityId: null
+        reportingFacilityIdType: null
+        suppressQstForAoe: false
+        suppressHl7Fields: "SPM-2-1"
+        suppressAoe: false
+        defaultAoeToUnknown: false
+        replaceUnicodeWithAscii: false
+        useBlankInsteadOfUnknown: null
+        truncateHDNamespaceIds: false
+        truncateHl7Fields: null
+        usePid14ForPatientEmail: false
+        convertTimestampToDateTime: null
+        cliaForOutOfStateTesting: null
+        cliaForSender: {}
+        phoneNumberFormatting: "STANDARD"
+        suppressNonNPI: false
+        processingModeCode: null
+        replaceDiiWithOid: null
+        applyOTCDefault: false
+        useOrderingFacilityName: "STANDARD"
+        valueSetOverrides: {}
+        nameFormat: "STANDARD"
+        receivingOrganization: null
+        convertPositiveDateTimeOffsetToNegative: false
+        stripInvalidCharsRegex: null
+        convertDateTimesToReceiverLocalTime: false
+        useHighPrecisionHeaderDateTimeFormat: false
+        type: "HL7"
+        truncationConfig:
+          truncateHDNamespaceIds: false
+          truncateHl7Fields: []
+          customLengthHl7Fields: {}
+      jurisdictionalFilter:
+        - "(Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state = 'MN') or (Bundle.entry.resource.ofType(Patient).address.state = 'MN')"
+      qualityFilter:
+        - "((Bundle.entry.resource.ofType(Specimen).collection.collectedPeriod.exists() or Bundle.entry.resource.ofType(Specimen).collection.collected.exists()) or Bundle.entry.resource.ofType(serviceRequest).occurrence.exists() or Bundle.entry.resource.ofType(Observation).effective.exists())"
+        - "(Bundle.entry.resource.ofType(Patient).address.line.exists() or Bundle.entry.resource.ofType(Patient).address.postalCode.exists() or Bundle.entry.resource.ofType(Patient).telecom.exists())"
```

Comment:
Which previous settings that you are referring to?  I am looking at their prod setting and didn't see those settings.  Would you let me know?

Reply:
- "hasAtLeastOneOf(patient_street,patient_zip_code,patient_phone_number,patient_email)"
I think the following is good.  Because patient_email is in the .telecom resource portion:
- "(Bundle.entry.resource.ofType(Patient).address.line.exists() or Bundle.entry.resource.ofType(Patient).address.postalCode.exists() or Bundle.entry.resource.ofType(Patient).telecom.exists())"

Reply:
Ah, you are right. I forgot the email was included in the telecom. Thank you!